packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -194,7 +121,36 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
               </Box>
             </React.Fragment>
           );
-        })}
+        })
+      ) : (
+        <Card data-cy="tokens-empty-state">
+          <Stack orientation="horizontal" spacing="space70">
+            <Box
+              as="img"
+              src="/images/patterns/empty-no-results-found.png"
+              alt="No results found illustration"
```

Comment:
Nit (non-blocking): This image is decorative because it doesn't add anything that the text on the page doesn't already say. You can just have alt="".